schedule/yast/encryption/activate_encrypted_volume+import_users_pvm.yaml

```diff
@@ -25,7 +25,6 @@ schedule:
   - installation/partitioning_finish
   - installation/installer_timezone
   - installation/user_import
-  - installation/user_settings_root
```

Comment:
@jknphy Same here, do we need this change, or we should replace it here with the new test module? (`root_simple_pwd.pm`)